packages/react-native-platform-override/src/Override.ts

```diff
@@ -0,0 +1,299 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ *
+ * @format
+ */
+
+import * as Serialized from './Serialized';
+import * as path from 'path';
+
+import UpgradeStrategy, {UpgradeStrategies} from './UpgradeStrategy';
+import ValidationStrategy, {ValidationStrategies} from './ValidationStrategy';
+import OverrideFactory from './OverrideFactory';
+
+/**
+ * Immmutable programatic representation of an override. This should remain
+ * generic to files vs directories, different representations, different
+ * validation rules, etc.
+ */
+export default interface Override {
+  /**
+   * Identifer of the override (e.g. filename or directory name)
```

Comment:
When looking in Manifest.ts, maybe add a comment that the identifier is case sensitive?